src/modules/ttlmonitor/TTLMonitorInstrument.cpp

```diff
@@ -101,73 +104,39 @@ void TTLMonitorInstrument::processMonitorReadouts(void) {
       continue;
     }
 
-    if (readout.BC != 0) {
-      XTRACE(DATA, WAR, "BC (%u) must be zero", readout.BC);
-      counters.MonitorErrors++;
-      continue;
-    }
-
-    if (readout.OTADC != 0) {
-      XTRACE(DATA, WAR, "OTADC (%u) must be zero", readout.OTADC);
-      counters.MonitorErrors++;
-      continue;
-    }
-
-    if (readout.GEO != 0) {
-      XTRACE(DATA, WAR, "GEO (%u) must be zero", readout.GEO);
-      counters.MonitorErrors++;
-      continue;
-    }
-
-    if (readout.TDC != 0) {
-      XTRACE(DATA, WAR, "TDC (%u) must be zero", readout.TDC);
-      counters.MonitorErrors++;
-      continue;
-    }
-
-    if (readout.VMM != 0) {
-      XTRACE(DATA, WAR, "VMM (%u) must be zero", readout.VMM);
-      counters.MonitorErrors++;
-      continue;
-    }
 
     uint32_t PixelId = 1;
     XTRACE(DATA, DEB, "Pixel: %u TOF %" PRIu64 "", PixelId, TimeOfFlight);
-    if (UseEveryNEvents == 1) {
```

Comment:
We needed to be able to reduce the number of events we were counting using the UseEveryNEvents parameter here, has that been replaced with something else, or are we losing that functionality?

Reply:
Did we? I Recall adding this functionality initially but Tobias seemed to be against so I removed it.